src/tests-bns/event-server-tests.ts

```diff
@@ -1027,9 +1036,55 @@ describe('BNS event server tests', () => {
     expect(namespaceList.results).toStrictEqual(['ape.mega']);
   });
 
-  afterEach(async () => {
-    await eventServer.closeAsync();
-    await db?.close();
-    await runMigrations(undefined, 'down');
+  test('If there is an event request error, then the event will not be recorded in the events_observer_request table', async () => {
```

Comment:
This looks great. Can you just move it to the Event Replay test suite instead of BNS?

Reply:
Ok